k8s/templates/prober-service.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "studio.fullname" . }}-prober-service
+  labels:
+    app: {{ template "studio.fullname" . }}-prober
+spec:
+  ports:
+  - port: 80
+    targetPort: {{ .Values.studioProber.port }}
+  selector:
+    app: {{ template "studio.fullname" . }}-prober
+  type: NodePort
```

Comment:
I didn't catch this -- what's the reason you chose a node port here?

Reply:
oh sorry I was unsure about this initially and was thinking about looking into it later but I definitely forgot about it... It's better to remove it, I guess? Thanks!